src/components/about.scss

```diff
@@ -0,0 +1,5 @@
+$blue: #bb1212;
```

Comment:
Maybe we shouldn't include .scss files in our commit since they are already outputting .css files. What do you think? @Danitlls

Reply:
I feel more comfortable leaving both there. beso

Reply:
❤️